packages/@jsii/dotnet-runtime/src/Directory.Build.targets

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.5.0" />
+    <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="3.1.3" />
+    <PackageReference Update="Microsoft.Extensions.Logging" Version="3.1.3" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="3.1.3" />
+
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Update="NSubstitute" Version="4.2.1" />
+    <PackageReference Update="xunit" Version="2.4.1" />
+    <PackageReference Update="xunit.runner.visualstudio" Version="2.4.1" />
+    <DotNetCliToolReference Update="dotnet-xunit" Version="2.3.1" />
+
+    <PackageReference Update="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Should you drop the Version property here, just like in the other files?

Reply:
This is the place that actually specifies the versions for all of them (which is why `Update` instead of `Include`). This is how dotnet has you set up solution-wide version configuration for your dependencies.